### ISSUE TITLE
VoiceOver accessibility pass

### ISSUE THREAD

**Issue by fionawhim** [CityOfBoston/311#359]
A lot of the main page is inaccessible because of heading levels and labels not lining up, and a lack of "skip navigation"
